### PR TITLE
#173766432 Email validation

### PR DESCRIPTION
**Pull request by mbabazisuzanne**
#### What does this PR do?
Validate user name to accept emails only.

#### Description of the task to be completed
POST api/users/signup

#### How should this manually be tested?
Using the above endpoint to test it in Postman

### CODE REVIEW

**Comment by Roland-Sankara** [comment]
LGTM

File: src/server/controllers/user.js [hunk 1]
```diff
@@ -1,106 +1,144 @@
-const User = require('../models/userModel');
```
Comment (3Nakajugo) [inline src/server/controllers/user.js]: @mbabazisuzanne , please do not make changes to the entire file. Only make changes to the necessary lines of code .

**Review by 3Nakajugo — DISMISSED** [review]
Please rebase your pull request

File: src/server/controllers/user.js (hunk 1 above)
Comment (davidgoodson) [reply src/server/controllers/user.js]: @3Nakajugo cc @mbabazisuzanne , I think this needs to be changed entirely, I looked at the code and the validation was put in the controller, email validation must be made in the Joi validation schema using regular expression.

**Comment by davidgoodson** [comment]
LGTM

**Review by davidgoodson — DISMISSED** [review]
LGTM

**Review by davidgoodson — COMMENTED** [review]
LGTM

**Pull request merged**